examples/region_of_interest/ExtractLineProfile.py

```diff
@@ -0,0 +1,57 @@
+"""
+Extract line profile from image
+===============================
+
+Use a ``Line2DROI`` to interactively extract a line profile with a certain width
+from an image.
+
+"""
+
+#%%
+# Initialize image data as HyperSpy signal:
+import hyperspy.api as hs
+import scipy
+img = hs.signals.Signal2D(scipy.datasets.ascent())
+
+#%%
+# Intialize Line-ROI from pixel (90,90) to pixel (200,250) of width 5.
+# You can also use calibrated axes units by providing floats instead of integers.
+line_roi = hs.roi.Line2DROI(90, 90, 200, 250, 5)
+
+#%%
+# Plot the image and display the ROI (creates new signal object):
+img.plot(cmap='viridis')
+roi1D = line_roi.interactive(img, color='yellow')
+
+#%%
+# You can drag and drop the ends of the ROI to adjust.
+# Print the (updated) parameters of the ROI:
+print('%.3f, %.3f, %.3f, %.3f, %.2f' % (line_roi.x1, line_roi.y1, line_roi.x2, line_roi.y2, line_roi.linewidth))
```

Comment:
```suggestion
print(line_roi)
```

Reply:
I use the complicated version, because then I can paste the string back into the initialization after interactive adjustment, but then I would need to add an extra comment.

Reply:
Ok, maybe use the following then to get the list of argument values:

```python
print(tuple(line_roi))
```